modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1180,6 +1190,34 @@ private ObjectFieldSetting _createObjectFieldSetting(
 		return objectFieldSetting;
 	}
 
+	private void _enableOracleSyntax() {
+		Props props = new PropsImpl();
+
+		String jdbcURL = props.get("jdbc.default.url");
```

Comment:
it is better to move this inside the if statement

Reply:
updated! thank you